packages/faceted-search/src/service/CRUDBadges.js

```diff
@@ -0,0 +1,61 @@
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
This file do not look like a service. may be just a badge.js on the package root.

Reply:
What do you define by service ?